generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -73,6 +73,12 @@ def __init__(self, frequency, reference_value_type, reference_value):
         self.reference_value_type = enums.LCRReferenceValueType(reference_value_type)
         self.reference_value = reference_value
 
+    def create_copy(self, target_class):
```

Comment:
I suppose here you copy members from one type to another type that has the same `__init__` interface.
Customers shouldn't call this, right? If so add a prefix `_`.

Reply:
Yeah, this is just called in `metadata_add_all.py`, so I wouldn't expect customers to have a need for it. I've renamed it `_create_copy()`.